.github/workflows/weekly.yml

```diff
@@ -30,23 +30,16 @@ jobs:
     strategy:
       matrix:
         flink_branches: [ {
-          flink: 1.18-SNAPSHOT,
+          flink: 2.1-SNAPSHOT,
+          jdk: '11, 17, 21',
           branch: main
-        }, {
-          flink: 1.19-SNAPSHOT,
-          jdk: '8, 11, 17, 21',
-          branch: main
-        }, {
-            flink: 1.20-SNAPSHOT,
-            jdk: '8, 11, 17, 21',
-            branch: main
-          }, 
+        }, 
         {
           flink: 1.20,
           jdk: '8, 11, 17, 21',
           branch: main
         }, {
-          flink: 1.19.0,
+          flink: 1.19.2,
           jdk: '8, 11, 17, 21',
           branch: v1.2
         }, {
```

Comment:
sorry, I think I missed it:
since 1.18-SNAPSHOT was removed in this PR
how about replacing 1.18.2 (which is a couple of lines below) with 1.20.1 ?

Reply:
We already run against 1.20 (https://github.com/apache/flink-connector-opensearch/pull/50/files#diff-b077dee47bccc37c2b6fef32cdd102f36acadad73ede1f410c3941f39c50cd94R38), do you mean replacing `1.20` with `1.20.1`?